src/api/QdmCalculationService.ts

```diff
@@ -1,17 +1,22 @@
 import { Calculator } from "cqm-execution";
-import cqmMeasure from "../mockdata/qdm/CMS108/cqm_measure.json";
-import valueSets from "../mockdata/qdm/CMS108/value_sets.json";
-import patient from "../mockdata/qdm/CMS108/IPP_DENOME_NUMER_PASS_NoVTEPatientRefusal.json";
+import { CqmMeasure, ValueSet } from "cqm-models";
 
 export class QdmCalculationService {
-  async calculateQdmTestCases() {
+  async calculateQdmTestCases(
```

Comment:
Although we have very basic cqmMeasure, a couple of tests would be good. we can keep updating/adding.

Reply:
I have just added test cases